dhis-2/dhis-support/dhis-support-commons/src/main/java/org/hisp/dhis/commons/util/TextUtils.java

```diff
@@ -320,19 +321,12 @@ public static String getString( String string, boolean emptyString )
      */
     public static String getCommaDelimitedString( Collection<?> elements )
     {
-        final StringBuilder builder = new StringBuilder();
-
-        if ( elements != null && !elements.isEmpty() )
+        if ( elements != null )
         {
-            for ( Object element : elements )
-            {
-                builder.append( element.toString() ).append( DELIMITER );
-            }
-
-            return builder.substring( 0, builder.length() - DELIMITER.length() );
+            return elements.stream().map( Object::toString ).collect( Collectors.joining( DELIMITER ) );

```

Comment:
I'd add a simple test after this modification, something like:
```
  @Test
    public void testGetCommaDelimitedString()
    {
        assertThat( TextUtils.getCommaDelimitedString( Arrays.asList( 1, 2, 3, 4, 5 ) ), is( "1, 2, 3, 4, 5" ) );
        assertThat( TextUtils.getCommaDelimitedString( Collections.singletonList( 1 ) ), is( "1" ) );
        assertThat( TextUtils.getCommaDelimitedString( null ), is( "" ) );
    }
```

Reply:
Added into tests.